src/components/ProjectList/listOfProjects.js

```diff
@@ -488,6 +488,24 @@ const projectList = [
     githubLink: 'https://github.com/ows-ali/Hacktoberfest/',
     description: 'Opportunity to start open source and PR experience for beginners',
     tags: ['hacktoberfest','hackerrank','low-hanging-fruit']
+  },
+  {
+    name: 'HoodieCamp',
+    githubLink: 'https://github.com/hoodiehq/camp/',
+    description: 'A space to welcome new and existing contributors.',
+    tags: ['first-timers-only']
+  },
+  {
+    name: 'Yarn',
+    githubLink: 'https://github.com/yarnpkg/yarn/',
+    description: 'Fast, reliable, and secure dependency management.',
+    tags: ['good first issue']
```

Comment:
I guess tags could be improved for all these new items.

For example, for yarn, tags could be `javascript`, `package manager` etc